tests/unit_test/app_opt/quantization/quantization_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+from nvflare.apis.dxo import DXO, DataKind
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_opt.quantization.model_dequantizor import ModelDequantizor
+from nvflare.app_opt.quantization.model_quantizor import ModelQuantizor
+
+TEST_CASES = [
+    (
```

Comment:
Can you add tests for float16 as well?

Reply:
the first test is for float16, and second for block8 (middle line for quant mode)

Reply:
check bitsandbytes' offerings (4 bit), provide api for take additional quantizor / dequantizor